### DISCUSSION TITLE
Jaxlib 0.3.22-cp310 for OSX 13, M1... "not a supported wheel on this platform"

### DISCUSSION THREAD

**Discussion by vputz** [jax-ml/jax#13030] [Q&A]
I have a developer trying to use jax/jaxlib on a Mac M1, OSX 13.0.  The provided pypi wheels (https://files.pythonhosted.org/packages/b6/ed/99dcb2997b3aeb76cace027db9fa38b4e8ca573273836b8ad021e0390d3f/jaxlib-0.3.22-cp310-cp310-macosx_11_0_arm64.whl) result in the message `not a supported wheel on this platform` for him.

I tried having him compile from source, but bazel gave him a few problems with inclusions and protobuf.  My next try is going to be cross-compiling from my linux box, but I'm not sure how much faith I have there.

Any suggestions on how to proceed?

**Comment by hawkinsp** [comment]
What does
```
python -c "import platform; print(platform.python_version(), platform.machine(), platform.python_implementation())"
```
print on that machine?

How was Python installed?

**Comment by vputz** [comment]
OK, actually very interesting; it's reading `3.10.0 x86_64 CPython` and was installed through poetry, but that value for python.machine doesn't make sense.  This may be a strange conda problem as well; this venv was made with poetry but he may have installed python itself via conda.  I'm trying to sort out what's going on.
